### DISCUSSION TITLE
Youtube Shorts Page

### DISCUSSION THREAD

**Discussion by tourscale** [jordwest/news-feed-eradicator#443] [Ideas]
[](url)All of us can relate to reel-style videos being very addicting. I love when I go to youtube.com Newsfeed Eradicator eradicates the homepage.... but all I do is click on shorts and start to binge.

I would love to get this eradicated. Buying the dev a coffee and leaving screenshots here.
<img width="1506" alt="Screenshot 2022-12-07 at 12 13 55 PM" src="https://user-images.githubusercontent.com/120050476/206263086-6bc888e7-476d-424c-bf44-89746316f3b9.png">
<img width="1486" alt="Screenshot 2022-12-07 at 12 13 48 PM" src="https://user-images.githubusercontent.com/120050476/206263091-98a6a91c-413f-4abd-a83a-c38cade66c2d.png">

**Comment by EliseiNicolae** [comment]
up

**Comment by rattrayalex** [comment]
"[Saner Social Media](https://github.com/tobiasdalhof/sanersocialmedia)" is an alternative chrome extension, also open-source, which claims to support this.

**Comment by maribox** [comment]
> "[Saner Social Media](https://github.com/tobiasdalhof/sanersocialmedia)" is an alternative chrome extension, also open-source, which claims to support this.

Yes but not available on Firefox
